lib/wiki_assignment_output.rb

```diff
@@ -130,8 +129,7 @@ def replace_or_add_assignment_tag(page_content, existing_tag, new_tag)
     return page_content if page_content.include?(new_tag_with_signature)
 
     # Otherwise, we need to add the tag to the right place.
-    page_content = insert_tag_into_talk_page(page_content, new_tag_with_signature)
-    page_content
+    insert_tag_into_talk_page(page_content, new_tag_with_signature)
```

Comment:
Keep this-as, since it makes it clear it more clear that the final return value is a further iteration of the `page_content` that is being modified throughout the method.